### ISSUE TITLE
Geocoder

### ISSUE THREAD

**Issue by klinux** [jsvd/logstash-filter-geocoder#1]
This plugin is still in development? Or it's dead?

Thank you.

**Comment by jsvd** [comment]
this is something I've made as an experiment, but I haven't had a need to develop it further.
What did you have in mind?

**Comment by klinux** [comment]
Yes, I have that need, I have to get lat and long of addresses in a list, so this plugin would be usefull to me.

**Comment by pemontto** [comment]
+1
